extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+
+#### bufferView
+The `bufferView` contains the compressed data.
+
+
+Below is an example of what a part of a glTF file my look like if the Draco extension is set:
+
+```javascript
+{
+    "extensionsRequired" : [
+        "Draco_animation_compression"
+    ],
+    "extensionsUsed" : [
+        "Draco_animation_compression"
+    ],
+    "extensions" : {
+        "Draco_animation_compression" : {
+            "compressedAnimations" : [
+                {
+                    "input" : 0,
+                    "outputs" : [ 1, 2, 3 ],
+                    "attributeId" : [ 1, 2, 3 ],
+                    "bufferView" : 0
+                },
+                {
+                    "input" : 4,
+                    "outputs" : [ 5 ],
+                    "attributeId" : [ 1 ],
+                    "bufferView" : 1
+                }
+            ],
+        }
+    },
+    "nodes" : [
+      ...
+    ],
+    "animations" : [
+        {
+            "channels" : [
+                {
+                    "sampler" : 0,
+                    "target" : {
+                        "node" : 0,
+                        "path" : "translation"
+                    }
+                },
+                {
+                    "sampler" : 1,
+                    "target" : {
+                        "node" : 1,
+                        "path" : "rotation"
+                    }
+                },
+            ],
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 1
+                },
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 2
+                }
+            ]
+        },
+        {
+            "channels" : [
+                {
+                    "sampler" : 0,
+                    "target" : {
+                        "node" : 2,
+                        "path" : "rotation"
+                    }
+                },
+                {
+                    "sampler" : 1,
+                    "target" : {
+                        "node" : 3,
+                        "path" : "translation"
+                    }
+                },
+            ],
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 3
+                },
+                {
+                    "input" : 4
+                    "interpolation" : "LINEAR",
+                    "output" : 5
+                }
+            ]
+        },
+    ],
+    "accessors" : [
+        // Accessors of input/output of animations that use the compression extension will omit "bufferView" property.
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
```

Comment:
Consider this a strawman idea, but could this extension also be structured:

```
{
    "extensionsRequired" : [
        "Draco_animation_compression"
    ],
    "extensionsUsed" : [
        "Draco_animation_compression"
    ],
    "extensions" : {
        "Draco_animation_compression" : {
            "compressedBufferViews" : [ 0, 1 ]
        }
    },
    "nodes" : [
      ...
    ],
    "animations" : [
      ... unmodified glTF animations ...
    ],
    "accessors" : [{
       "componentType" : 5123,
       "count" : 100,
       "type" : "SCALAR",
       "extensions" : {
          "Draco_animation_compression" : {
            "bufferView": 0,
            "attributeId": 1
           }
        }
     }],
    ...
}
```

Reply:
So I think this should work. The one issue is that using this format the gltf file will usually be bigger. I ran through some test files and I saw anywhere from a 1-27% increase in size (unzipped). CessiumMan and Monster saw increases of 9% in gltf size.  About a 2.5 and 4% total size increase with mesh and animation compression turned on.

Reply:
Oh yikes, that's 9% of total file size occupied by JSON in the `accessors` array with this layout? Difference might disappear when served with GZIP, but still not ideal. Or were you grouping accessors into compressed buffer views differently with this layout, e.g. one animation compression per accessor?

Reply:
I wasn't grouping them differently, but I'm assumign every compressed animation accessor needs to have a bufferView and and attributeId (or I guess in case of input no attributeID, which should bring the size down a little)

Here is a link to what I think you are proposing:
https://drive.google.com/open?id=14_vRPj1Xd6X9TPHcPQkBCwbWHvzuKRah

 Ignore the actual values as I just put place holders in to get the size.